release-deployment.md

```diff
@@ -127,22 +130,21 @@ There's nothing special about that. Each developer follows the above [Develop a
    * Description: Include a high-level list of things changed in this release.
    Click `Publish release`.
 
-1. Merge the `release-vX.Y.Z` into `develop`.
+1. Merge `master` into `develop`.
```

Comment:
After the last two steps have been followed, `master` should be up to date with the release PR's changes. As previously discussed, `master` should update `develop` so that any hotfixes are caught and included in the downstream update.